README.md

```diff
@@ -2,8 +2,29 @@
 
 ## Description:
  
-To better support Instructors of Large Courses in Canvas, When we are given a CourseId we should be able create groups in Canvas based on their course sections. This will allow them to: 
-1) Share select resources with a section 
-2) Discussions available to specific section 
-3) Content Pages with a specific section 
-4) Conference with a specific section 
+To better support Instructors of Large Courses in Canvas,  given a CourseId we should be able create groups in Canvas based on their course sections. This will allow them to: 
+
+1. Share select resources with a section 
+2. Discussions available to specific section 
+3. Content Pages with a specific section 
+4. Conference with a specific section 
+
+This utility is written in python and takes `:course_id`, `:group_category_name` from the property file and creates a group category with the name provided in the property file. We will get all the sections and users
+in the course and create one group for each section and add all the users in each section to the corresponding group. The group name will be identical to the section name
+ 
+## Requirements 
+
```

Comment:
I have learned that if you do `pip install --user <module-name>` this will download the module to the users home directory that is on the python `sys.path` which is  is `~/Library/Python/2.7/lib/python/site-packages`. You can also install the modules to the applications specific directory by doing `pip install --user -r requirements.txt` and also set the environment variable as `PYTHONUSERBASE=/path/to/download/modules`. For [more info](https://pip.pypa.io/en/stable/user_guide/#user-installs) on this. The [Discussion](http://stackoverflow.com/questions/2915471/install-a-python-package-into-a-different-directory-using-pip) on this was useful.  I did not choose the top most answer from the discussion instead i felt going by `--user` option sounded elegant. I have added these notes in the readme as well